src/box/execute.c

```diff
@@ -629,3 +635,137 @@ sql_prepare_and_execute(const char *sql, int len, const struct sql_bind *bind,
 	port_destroy(port);
 	return -1;
 }
+
+/** Table of methods of SQL user-defined functions. */
+static struct func_vtab func_sql_expr_vtab;
```

Comment:
The files execute.h/.c don't seem as an appropriate place for such unrelated functionality. They are not related to `box.execute` neither to `IPROTO_EXECUTE`, are they? I would advise to move them somewhere into sql/ folder. For instance, create sql/func.h and add them to func.c/.h there. If you need `port_sql`, then it could be moved as well.

Reply:
Thanks. Moved `struct port_sql` from `box/execute` to sql/port` and moved new functionality to `sql/func`.